frontend/src/components/advanced-auto-filter/advanced-auto-filter.tsx

```diff
@@ -236,6 +243,19 @@ const AdvancedAutoFilter: FC = () => {
         <UTurnRightIcon className={styles.resetIcon} />
         Reset All Filters
       </h6>
+      {isButtonVisible && (
+        <Zoom in={isButtonVisible}>
+          <div className={styles.searchButtonWrapper}>
+            <Button
+              onClick={doSearch}
+              className={styles.searchButton}
+              variant="contained"
+            >
+              SEARCH
+            </Button>
+          </div>
+        </Zoom>
+      )}
```

Comment:
`isButtonVisible &&` will make the `Zoom` immediately unmounted and the out animation won't work